src/features/internal/cheatcodes/pages/NavigationSubscription/NavigationSubscription.tsx

```diff
@@ -52,6 +53,13 @@ export function NavigationSubscription(): React.JSX.Element {
     <ScrollView>
       <PageHeaderSecondary title="Subscription 🔔" />
       <StyledContainer>
+        <Row half>
+          <InternalTouchableLink
+            as={ButtonPrimary}
+            wording="Ecran de séléction"
```

Comment:
C'est les cheatcodes dont on s'en fout mais au cas ou : 
```suggestion
            wording="Ecran de sélection"
```